docs/user-guide/tasks/http.md

```diff
@@ -19,19 +20,34 @@ iter8 k launch \
+--set assess.SLOs.upper.http/postit/latency-mean=150 \
+--set assess.SLOs.upper.http/postit/error-count=0 \
+--set runner=job
+```
+
 ## Parameters
 
 | Name | Type | Description |
 | ---- | ---- | ----------- |
-| url  | string (URL) | HTTP URL where requests are sent. |
-| headers  | map[string]string | HTTP headers to use in the requests. |
-| numRequests  | int | Number of requests to be sent to the app. Default value is 100. |
-| duration  | string | Duration of this task. Specified in the [Go duration string format](https://pkg.go.dev/maze.io/x/duration#ParseDuration) (example, `5s`). If both duration and numRequests are specified, then duration is ignored. |
-| qps  | float | qps stands for queries-per-second. Number of requests per second sent to the app. Default value is 8.0. |
-| connections  | int | Number of parallel connections used to send requests. Default value is 4. |
-| payloadURL  | string (URL) | URL from which to download the content that will be used as the request payload. If this field is specified, Iter8 will send HTTP POST requests to the app using this content as the payload. |
-| payloadStr  | string | String data to be used as the request payload. If this field is specified, Iter8 will send HTTP POST requests to the app using this string as the payload. |
-| contentType  | string | Content type of the payload. This is intended to be used in conjunction with one of the `payload*` fields. If this field is specified, Iter8 will send HTTP POST requests to the app using this as the Content-Type header value. |
+| url | string (URL) | URL where requests are sent. |
+| headers | map[string]string | HTTP headers to use in the requests. |
+| numRequests | int | Number of requests to be sent to the app. Default value is 100. |
+| duration | string | Duration of this task. Specified in the [Go duration string format](https://pkg.go.dev/maze.io/x/duration#ParseDuration) (example, `5s`). If both duration and numRequests are specified, then duration is ignored. |
+| qps | float | qps stands for queries-per-second. Number of requests per second sent to the app. Default value is 8.0. |
+| connections | int | Number of parallel connections used to send requests. Default value is 4. |
+| payloadURL | string (URL) | URL from which to download the content that will be used as the request payload. If this field is specified, Iter8 will send HTTP POST requests to the app using this content as the payload. |
+| payloadStr | string | String data to be used as the request payload. If this field is specified, Iter8 will send HTTP POST requests to the app using this string as the payload. |
+| contentType | string | Content type of the payload. This is intended to be used in conjunction with one of the `payload*` fields. If this field is specified, Iter8 will send HTTP POST requests to the app using this as the Content-Type header value. |
+| endpoints | map[string]interface | Used to test multiple HTTP endpoints. |
```

Comment:
How about 

`endpoints | map[string]EndPoint | Used to specify multiple endpoints and their configuration. `string` is the name of the endpoint. EndPoint struct includes all the parameters described above. Load testing will be performed and metrics will be collected for each endpoint separately. |

### Precedence
Describe how endpoint-specific values, common-values, and default-values take precedence in that order.